include/ei_ncs.rst

```diff
@@ -0,0 +1,136 @@
+
+Dependencies
+------------
+
+Before using the wrapper, you must prepare and deploy the machine learning model for your embedded device.
+
+Preparing machine learning model
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For information about building your own machine learning model refer to `Edge Impulse getting started guide`_.
+
+The machine learning model is distributed as a single zip archive with C++ library sources.
+The mentioned zip file is used by NCS build system to build the library.
+
+Additinal Kconfig requirements
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The C++ library downloaded from Edge Impulse studio requires enabling the following Kconfig options:
+
+* :option:`CONFIG_CPLUSPLUS`
+* :option:`CONFIG_STD_CPP11`
+* :option:`CONFIG_LIB_CPLUSPLUS`
+* :option:`CONFIG_NEWLIB_LIBC`
+* :option:`CONFIG_NEWLIB_LIBC_FLOAT_PRINTF`
+* :option:`CONFIG_FPU`
+
+Building model in NCS
+~~~~~~~~~~~~~~~~~~~~~
+
+Enable building the downloaded library using :option:`CONFIG_EDGE_IMPULSE` option and specify the :option:`CONFIG_EDGE_IMPULSE_URI`.
+The URI can be:
+
+* An absolute path to file in local file system.
+  You must download the zip file manually and place it under path defined by the Kconfig option.
+* Any downloadable URI supported by CMake's ``file(DOWNLOAD)`` command.
+  The NCS build system will download the zip file automatically during build.
+  The zip file is downloaded into your application's build directory.
+  If the URI requires providing an additional API key, you can provide it using the following CMake definition: ``EI_API_KEY_HEADER``.
+  The API key is provided using a ``key_name:key_value`` format.
+
+Download model directly from Edge Impulse studio
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+After deploying the model, you can make the NCS build system download it directly from the Edge Impulse studio.
+Complete the following steps to do it:
+
+1. Set the CONFIG_EDGE_IMPULSE_URI to URI from Edge Impulse studio: ``CONFIG_EDGE_IMPULSE_URI="https://studio.edgeimpulse.com/v1/api/XXXXX/deployment/download?type=zip"``.
+   The ``XXXXX`` must be set to the project ID of your Edge Impulse project.
+   You can check the project ID of your project in project dashboard.
+
+#. Provide the ``x-api-key`` associated with your project.
+   The x-api-key is provided defining the ``EI_API_KEY_HEADER`` CMake option as ``x-api-key:aaaabbbbccccdddd``.
+   See :ref:`cmake_options` for more information about defining CMake options for command line builds and SEGGER Embedded studio.
+   You can access your API keys under ``Keys`` tab in the Edge Impulse project dashboard.
+
+Enabling the Edge Impulse NCS library
+-------------------------------------
+
+The Edge Impulse NCS library is enabled by :option:`CONFIG_EI_NCS` option.
+The option is enabled by default if :option:`CONFIG_EDGE_IMPULSE` is set.
+
+If you disable the :option:`CONFIG_EI_NCS`, you can directly use the API provided by Edge Impulse C++ library downloaded from Edge Impulse studio.
```

Comment:
I would remove this and the following sentence.

Reply:
done